client/src/tasks.rs

```diff
@@ -219,7 +254,22 @@ pub fn run_task(task: Task) {
                     rts.abort_task(id.to_string());
                 }
             }
-        },
+        }
+        NETWORK_SCAN_CMD => {
+            let ports: Vec<u16> = task.data.split(',')
```

Comment:
Does this only scan ports contained in the task.data property? It would perhaps be nice to support port ranges, for example port 25500-25600, and to support scanning all ports, by, for example, not specifying any task.data, or setting task.data to *.

Reply:
This was a good idea! 

Added port parsing similar to nmap:
```
"1-5, 6, 7" => 1,2,3,4,5,6,7
"-3" => 1,2,3
"65533-"  => 65533, 65534
```